source/docs/fedora_atomic_bare_metal_installation.md

```diff
@@ -66,9 +66,13 @@ Fedora 25 Atomic:
 
     # NOTICE: This will download the content from upstream; this will be very slow.
     # Create your own OSTree repo locally and mirror the content instead.
```

Comment:
maybe add a note that says this will download the latest release, not necessarily what was delivered on the ISO